packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -157,6 +163,54 @@ export default decorate([
       redirectToTaskPage() {
         this.props.router.push('/tasks')
       },
+      showDescription() {
+        const {
+          data: { public: _public },
+          name,
+        } = this.props
+        delete _public.any
+        alert(
+          name,
+          <div>
+            {isEmpty(_public) ? (
+              <p>{_('hubTemplateDescriptionNotAvailable')}</p>
+            ) : (
+              <div>
+                {EXCLUSIVE_FIELDS.filter(field => field in _public).map(
+                  field => (
+                    <div key={field}>
+                      <p>{_public[field]}</p>
+                      <hr />
+                    </div>
+                  )
+                )}
+                {map(
+                  omit(_public, EXCLUSIVE_FIELDS.concat(BANNED_FIELDS)),
```

Comment:
You can use the same idea as https://github.com/vatesfr/xen-orchestra/pull/4593/files#r335980177